www/tynk00/sp/resources/views/livewire/project-task-manager.blade.php

```diff
@@ -0,0 +1,168 @@
+<div>
+
+    <div class="p-2 shadow-lg rounded bg-dark mb-3">
+        <a href="#" class="btn btn-light" data-toggle="modal" data-target="#addTaskModal">Nový úkol</a>
+        <div class="float-right">
+            <input type="checkbox" class="btn-check" id="btn-check-outlined" autocomplete="off" wire:model="closedTasks">
+            <label class="btn btn-outline-light" for="btn-check-outlined">Zobrazit dokončené úkoly</label><br>
+
+        </div>
+    </div>
+
+
+
+
+
+    <table class="table table-striped mb-0">
+        <thead>
+            <tr>
+                <th class="text-end text-nowrap" style="width: 50px;"></th>
+                <th>Name</th>
+                <th>Project</th>
+                <th>Due Date</th>
+                <th class="text-end text-nowrap" style="width: 50px;"></th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach ($tasks as $task)
```

Comment:
tady se zobrazuji tasky ze vsech projektu?

Reply:
V této komponentě by se mělo zobrazovat pouze úkolu z projektu, ve kterém je uložen, dále mám ještě podobnou komponentu task-manager, kde jsou všechny úkoly ze všech projektů